packages/hebao_v1/contracts/modules/core/WalletFactory.sol

```diff
@@ -40,20 +40,14 @@ contract WalletFactory
 
     mapping(address => bytes32) blanks;
 
-    address        public immutable walletImplementation;
-    bool           public immutable allowEmptyENS; // MUST be false in production
-    ControllerImpl public immutable controller;
-    bytes32        public immutable DOMAIN_SEPERATOR;
+    bytes32             public immutable DOMAIN_SEPERATOR;
+    ControllerImpl      public immutable controller;
+    address             public immutable walletImplementation;
+    bool                public immutable allowEmptyENS; // MUST be false in production
 
-    struct ControllerCache
-    {
-        address             ensResolver;
-        ENSReverseRegistrar ensReverseRegistrar;
-    }
-
-    ControllerCache public controllerCache;
-
-    BaseENSManager public immutable ensManager;
+    BaseENSManager      private immutable ensManager;
+    address             private immutable ensResolver;
+    ENSReverseRegistrar private immutable ensReverseRegistrar;
```

Comment:
Why `private` here?

Reply:
WalletFactory.sol doesn't have any sub-class and this method will never be used by other contracts.

Reply:
Oh, I forgot to change the back. Was trying to compile to see if they are used by any other contracts.

Reply:
Changed to `public`.